src/app/pages/my-request-detail/my-request-detail.component.spec.ts

```diff
@@ -100,79 +109,102 @@ describe('MyRequestsComponent', () => {
     });
   });
 
+  const request: DataRequest = {
+    id: '1',
+    label: 'request 1',
+    domainType: CatalogueItemDomainType.DataModel,
+    status: 'unsent',
+  };
+
+  // const dataSchemas: DataSchema[] = [
+  //   {
+  //     schema: buildDataClass('schema 1'),
+  //     dataClasses: [
+  //       {
+  //         dataClass: buildDataClass('s1.class 1'),
+  //         dataElements: [
+  //           buildDataElement('s1.c1.element 1'),
+  //           buildDataElement('s1.c1.element 1'),
+  //         ],
+  //       },
+  //       {
+  //         dataClass: buildDataClass('s1.class 2'),
+  //         dataElements: [
+  //           buildDataElement('s1.c2.element 1'),
+  //           buildDataElement('s1.c2.element 1'),
+  //         ],
+  //       },
+  //     ],
+  //   },
+  //   {
+  //     schema: buildDataClass('schema 2'),
+  //     dataClasses: [
+  //       {
+  //         dataClass: buildDataClass('s2.class 1'),
+  //         dataElements: [
+  //           buildDataElement('s2.c1.element 1'),
+  //           buildDataElement('s2.c1.element 1'),
+  //         ],
+  //       },
+  //     ],
+  //   },
+  // ];
+
```

Comment:
A lot of this commented out code was part of a previous refactor. I'm keeping it for now with a TODO comment, just in case. Should be looked at as part of #258